content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,302 @@
+
+Click :guilabel:`New rule` and set the first field to :guilabel:`Email`. Then, select :guilabel:`is
+set` from the drop-down menu in the second field.
+
+These rules add only leads with an associated contact method to the report.
+
+.. _active-status:
+
+1. Active status
+~~~~~~~~~~~~~~~~
+
+Click the :icon:`fa-sitemap` :guilabel:`(Branch)` icon to the right of the `Phone is set` line, to
+add a new rule that branches from the rules above.
+
+Two horizontal sets of fields appear below a line showing :guilabel:`any of:` with a
+:icon:`fa-caret-down` :guilabel:`(down caret)` icon. This setting filters for records that match
+**any** of the rules contained inside. This uses the same logic as an OR (|) logical operator.
+
+Set the first field to :guilabel:`Active`. Then, select :guilabel:`is set` in the next field.
+
+Next, click the :icon:`fa-plus` :guilabel:`(New Rule)` button next to :guilabel:`Active is set` to
+create a new line of fields beneath it.
+
+Set the first field to :guilabel:`Active`. Then, select :guilabel:`is not set` in the next field.
+
+.. image:: lead_distribution_report/active-set.png
+  :align: center
+  :alt: Use Active to include active status in the report.
+
+This rule adds the activity status of the lead to the report.
+
+.. note::
+  Active status is an important filter to include when creating a lead distribution report because
+  it includes **all** leads regardless of won/lost or active/inactive status in the report. This
+  provides a comprehensive view of all the leads assigned to each sales member.
+
+Filter for quality leads
+------------------------
+
+The following additional conditions are provided as an example of a *good*, but *not comprehensive*,
+set of rules for finding quality leads. These filters should be applied on top of the
+:ref:`essential-filters` in the order specified to achieve a heavily-detailed filter.
+
+:ref:`referred-by`
+
+:ref:`source`
+
+:ref:`notes`
+
+:ref:`tags`
+
+:ref:`email`
+
+:ref:`salesperson`
+
+These conditions can be added, removed, or modified to best fit the desired information in the
+report.
+
+.. _referred-by:
+
+1. Referred by
+~~~~~~~~~~~~~~
+
+Click the :icon:`fa-plus` :guilabel:`(New Rule)` button next to :guilabel:`Active is not set` to
+create a new line of fields beneath it.
+
+Next, set the first field to :guilabel:`Referred By`. In the second field, select
+:guilabel:`contains` from the drop-down menu. In the last field, type `appointment`.
+
+.. image:: lead_distribution_report/referred-by.png
+  :align: center
+  :alt: Add referred by as a branch to filter appointments.
+
+This rule adds any leads that were referred by an appointment to the report.
+
+.. _source:
+
+2. Source
+~~~~~~~~~
+
+Click the :icon:`fa-plus` :guilabel:`(New Rule)` button next to the line for `Referred by
+appointment`.
+
+Next, set the first field to :guilabel:`Source`. Select :guilabel:`contains` from the drop-down menu
+in the second field. Type `livechat` in the third field.
```

Comment:
same feedback as Referred by:

`livechat` is a value that's stored in our internal Odoo database from leads that come from the **Live Chat** module. That value isn't going to be relevant for other databases that either a) don't use Live Chat module or b) don't have that set as their value (crazy example, a database could set their **Source** UTM for Live Chat app as `porkchop` if they wanted to).

You're on the right track by mentioning **Source** as an additional filter but are being too specific in the values which are only relevant to us.

My feedback for the following sections is similar: 
- the parameters are correct however the values are too specific / not useful to the customer (only useful for us internally) 
- I don't think we need the sections + :ref: links anyway, and can just point to _Quality leads report_ documentation instead as a **seealso**, which already includes all of the relevant information around additional quality lead filters to add.